recipes/gmsh/tests.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt.edit
+index 5650947..6cdc869 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt.edit
+@@ -1916,6 +1916,8 @@ if(NOT DISABLE_GMSH_TESTS)
+   file(GLOB_RECURSE TESTFILES
+        tutorial/*.geo demos/*.geo benchmarks/?d/*.geo benchmarks/extrude/*.geo
+        benchmarks/boolean/*.geo)
++  list(REMOVE_ITEM TESTFILES "${CMAKE_SOURCE_DIR}/demos/post_processing/lowmem_anim.geo")
```

Comment:
This seems inside an if statement, is that correct?

Reply:
Yes exactly, it tests if tests are enabled by cmake flags and then builds the list of tests to execute by make test.